modules/security-network-isolating.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-isolating_{context}"]
+= Isolating applications
+
+{product-title} enables you to segment network traffic on a single cluster to
+make multitenant clusters that isolate users, teams, applications, and
+environments from non-global resources.
+
+For example, to isolate a project network in the cluster, run:
+
+----
+$ oc adm pod-network isolate-projects <project1> <project2>
```

Comment:
this command only used on `multitetnant` plugin mode.  however, the project1 and project2 are isolated by default. in 4.x version .the default plugin mode is `networkpolicy`.  So guess if this part is still needed.

Reply:
I'm cutting the example and adding a link after this section to "Configuring network isolation using OpenShift SDN" in the OpenShift docs.